File: day95/homework/00.py
def multiply_by_two(numbers):
    new_list = []
    for num in numbers:
        new_list.append(num * 2)
    return new_list

print(multiply_by_two([1, 2, 3, 4]))  

numbers = []

for i in range(5):
    num = int(input("enter number: "))
    numbers.append(num)

numbers.reverse()
print(numbers)


def word_lengths(words):
    lengths = []
    for word in words:
        lengths.append(len(word))
    return lengths
print(word_lengths("apple",))  

students = []
for i in range(3):  
    name = input("name: ")
    grades = []

    for j in range(3):  
        grade = float(input(f"{name}-ს ქულა #{j+1}: "))
        grades.append(grade)

    avg = sum(grades) / len(grades)
    students.append((name, avg))
for student in students:
    print(f"{student[0]}  average {student[1]}")


def filter_long_words(words):
    new_list = []
    for word in words:
        if len(word) > 3:
            new_list.append(word)
    return new_list

print(filter_long_words(["cat", "book", "hi", "school"])) 

File: day160/homework/00.js
const number=[1,2,3,4]
for(num of number){
    console.log(num*3)
}
const numbers = [1, 2, 3, 4];

const squared = numbers.map(num => num * num);
console.log(squared);

const num = [1, 2, 3, 4, 5, 6];

const evens = num.filter(num => num % 2 === 0);
console.log(evens);